modules/pipeline/dbclient/op_pipeline.go

```diff
@@ -473,10 +482,19 @@ func (client *Client) PageListPipelines(req apistructs.PipelinePageListRequest,
 	}
 	pipelines, err := client.ListPipelinesByIDs(pagingPipelineIDs, needQueryDefinition, ops...)
 	if err != nil {
-		return nil, pagingPipelineIDs, -1, -1, err
-	}
-
-	return pipelines, pagingPipelineIDs, total, currentPageSize, nil
+		return &PageListPipelinesResult{
+			PagingPipelineIDs: pagingPipelineIDs,
+			Total:             -1,
+			CurrentPageSize:   -1,
```

Comment:
what means -1? why not use default value 0?

Reply:
I don't know. The old logic. I just changed the lower structure

Reply:
then, you should to know it. you are team member responsible for the pipeline.

Reply:
ok